src/Silakov.Homework4/Task.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
+
+namespace Silakov.Homework4
+{
+    /// <summary>
+    /// Задача.
+    /// </summary>
+    internal class Task
+    {
+        /// <summary>
+        /// Идентификатор задачи.
+        /// </summary>
+        private int _id;
+        /// <summary>
+        /// Название задачи.
+        /// </summary>
+        private string _name;
+        /// <summary>
+        /// Описание.
+        /// </summary>
+        private string _description;
+        /// <summary>
+        /// Статус задачи.
+        /// </summary>
+        private StatusTask _status;
+        /// <summary>
+        /// Дата создания.
+        /// </summary>
+        private DateTime _creationDate;
+        /// <summary>
+        /// Дата выполнения.
+        /// </summary>
+        private DateTime _executionDate;
+
+        private static int counter = 1;
+
+        Task()
+        {
+            Console.Write("Name: ");
+            _name = Console.ReadLine();
+            Console.Write("Description: ");
+            _description = Console.ReadLine();
+
+            _creationDate = DateTime.Now;
+            _id = counter;
+
+            counter += 1;
+        }
+
+        public void Edit(int id)
+        {
+            Console.Write("Name: ");
+            _name = Console.ReadLine();
+            Console.Write("Description: ");
+            _description = Console.ReadLine();
+        }
+
+        /// <summary>
+        /// Вывести информацию о задаче.
+        /// </summary>
+        public void OutputInformationAboutTask()
+        {
+            Console.Write("Id - ");
+            Console.ForegroundColor = ConsoleColor.Red;
+            Console.Write(_id.ToString() + "\n");
+            Console.ResetColor();
+            Console.WriteLine($"Name - {_name}");
+            Console.WriteLine($"Description - {_description}");
+            Console.WriteLine($"Status - {_status}");
+            Console.WriteLine($"Creation date - {_creationDate}");
+            if (_executionDate != DateTime.MinValue)
+            {
+                Console.WriteLine($"Execution date - {_executionDate}");
+            }
+            Console.WriteLine("--------------------");
+        }
+
+        /// <summary>
+        /// Изменить статус задачи.
+        /// </summary>
+        public void ChangeStatus()
+        {
+            int numStatus = 1;
```

Comment:
this can be evaluated without iterating through all values, please try to refactor